docs/source/reference/howto/wipe_persistent_data.rst

```diff
@@ -23,56 +23,28 @@ in the sequence given below, so you shouldn't proceed to next steps until the pr
 
 #. Add "wipe_environment_enabled: True" to `public.yml` file.
 
-
-#. Stop CommCare and close Postgres sessions.
+#. Stop CommCare
 
    .. code-block::
 
       $ cchq <env_name> service commcare stop
-      $ cchq <env_name> service postgresql stop
-      $ cchq <env_name> service postgresql status
-
 
-   If that doesn't stop Postgres and PgBouncer, and if Postgres is
-   running on the same machine as you are logged in on, you can call
-   ``service`` directly:
+#. Reset PostgreSQL and PgBouncer
 
    .. code-block::
 
-      $ sudo service pgbouncer stop
-      $ sudo service postgresql stop
-      $ sudo service postgresql start
-      $ sudo service pgbouncer start
+      $ cchq <env_name> ap deploy_postgres.yml
 
-#. Wipe PostgreSQL databases
+#. Wipe PostgreSQL data
 
-   Check status & start postgres if NOT running.
+   Check status. Once status is "OK", wipe PostgreSQL data
 
    .. code-block::
 
       $ cchq <env_name> service postgresql status
-
```

Comment:
What is the reason for removing this? Or is it simply that we're expecting the user to use the other parts of the docs for further debugging in case there's issues (which I think is OK since handling issues is probably out of scope here)

Reply:
> is it simply that we're expecting the user to use the other parts of the docs for further debugging in case there's issues

Yeah, pretty much this.

It turns out that the user will need to reset Postgres & PgBouncer anyway, so I removed these steps because they aren't useful.

(I'm opening follow-up tickets for commands given in this document, because there are at least two bugs (maybe more) that need to be fixed. So there will be more testing here, and this doc may change in the near future.)